6.x/crud-columns.md

```diff
@@ -1589,6 +1587,49 @@ $this->crud->addColumn([
 ```
 
 
+<a name="link-column-to-routes"></a>
+### Link Column To Route
+
+To make a column link to a route URL, you can use the `linkTo($routeName, $target = null)` helper. Behind the scenes, this helper will use the `wrapper` helper to set up a link towards the route you want. See the section above for details on the `wrapper` helper.
+
+It's dead-simple to use the `linkTo()` helper to point to a route name:
+```php
+// you can do:
+$this->crud->column('category')->linkTo('category.show');
+
+// instead of:
+$this->crud->column('category')->wrapper([
+    'href' => function ($crud, $column, $entry, $related_key) {
+        return backpack_url('category/'.$related_key.'/show');
+    },
+]);
+```
+
+You can also link to non-related urls, as long as the route has a name.
+
+```php
+$this->crud->column('my_column')->linkTo('my.route.name');
+
+// you can also open them in a new tab
+$this->crud->column('my_column')->linkTo('my.route.name', '_blank');
+
+// array syntax is also supported
+
+$this->crud->column([
+    'name' => 'category',
+    'linkTo' => 'category.show',
+
+    // alternatively with target configuration
+    'linkTo' => [
+        'nameName' => 'category.show',
```

Comment:
Heads-up, the docs said `name` here but I discovered what works is actually `routeName`. So I used that. I don't love it to be honest, I would rather have `route` here... what do you think?